.github/workflows/benchmarks_run.yml

```diff
@@ -29,7 +29,7 @@ jobs:
     env:
       IRIS_TEST_DATA_LOC_PATH: benchmarks
       IRIS_TEST_DATA_PATH: benchmarks/iris-test-data
-      IRIS_TEST_DATA_VERSION: "2.19"
+      IRIS_TEST_DATA_VERSION: "2.21"
```

Comment:
Not sure why it's 2 versions, but we're definitely now on 2.21
https://github.com/SciTools/iris-test-data/commit/a2e2bb7eacadf9ba77ca32e25a4a125f022b85ee

Reply:
Looks like `v2.20` was to help [ncdata](https://github.com/pp-mo/ncdata) - SciTools/iris-test-data#85 - rather than it being needed for any changes within Iris.